cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -19,6 +19,9 @@ spec:
         version: v11
         kubernetes.io/cluster-service: "true"
     spec:
+{% if grains['cloud'] is defined and grains['cloud'] in [ 'vsphere', 'photon-controller' ] %}
```

Comment:
@AlainRoy as a general problem with this integration: your pods need to be able to do DNS lookups beyond the cluster, or they will likely be unhappy.  Running your DNS server as `hostNetwork` seems like a bad answer to me - you will be doing DNS lookups for hosts that pods (ostensibly) can't reach.

We can either configure Kube-DNS to have an arbitrary upstream DNS (e.g. 8.8.8.8) or you can configure it to not have upstream DNS or you can find a way to bridge your pod network and your host network.  It is not strictly a requirement that Pods be able to DNS-resolve anything outside of the cluster (though the impl might demand it, that should be fixed).  This is a policy decision for you, the owner of the integration.

Do you believe that all users of Photon will be ok with DNS not working except for in-cluster names?